clustered/integration-test/src/test/java/org/ehcache/clustered/IterationFailureBehaviorTest.java

```diff
@@ -120,8 +120,8 @@ public void testIteratorFailover() throws Exception {
       } catch (CacheIterationException e) {
         assertThat(e.getCause(), instanceOf(StoreAccessException.class));
         assertThat(e.getCause().getCause(), instanceOf(ServerStoreProxyException.class));
-        assertThat(e.getCause().getCause().getCause().getCause(),
-          either(instanceOf(ConnectionClosedException.class)) //lost in the space between active and passive
+        assertThat(e.getCause().getCause().getCause(),
+          either(instanceOf(ReconnectInProgressException.class)) //lost in the space between active and passive
```

Comment:
Can you explain the need for this change?

Reply:
So does `ConnectionClosedException` no longer appear in the exception cause chain?  If not, maybe it should.

Reply:
@chrisdennis Is there any reason for the `ReconnectInProgressException` used by `ReconnectInProgressClusterTierClientEntity` include the exception causing the reconnect attempt, i.e. the `ConnectionClosedException`?